File: samples_generated_code/samples_1/qwen/baseline_code_java/code_row_106.java
import java.util.Scanner;

public class SalesCalculator {
    public static void main(String[] args) {
        double totalSales;
        totalSales = 0.0 + 12345.67; // January sales
        totalSales += 45678.90;       // February sales
        totalSales += 67890.12;       // March sales
        System.out.println("Total Sales: " + totalSales);
    }
}

File: samples_generated_code/sample_3/phi4/baseline_code_c/code_row_106.c
#include <stdio.h>

int main() {
    double totalSales = 0;
    
    totalSales += 1234.56; // Add January sales
    totalSales += 7890.12; // Add February sales
    totalSales += 3456.78; // Add March sales
    
    printf("Total Sales: %.2f\n", totalSales);
    return 0;
}